src/Markdig.Wpf/Renderers/Wpf/Extensions/TableRenderer.cs

```diff
@@ -65,7 +65,7 @@ protected override void Write([NotNull] WpfRenderer renderer, [NotNull] Table ta
                     renderer.Write(cell);
                     renderer.Pop();
 
-                    if (table.ColumnDefinitions != null)
+                    if (table.ColumnDefinitions != null && table.ColumnDefinitions.Count > 0)
```

Comment:
```suggestion
                    if (table.ColumnDefinitions.Count > 0)
```
`table.ColumnDefinitions` is guaranteed to be non-null at this point.